device-posture-api/index.js

```diff
@@ -0,0 +1,15 @@
+const list = document.querySelector("ul");
+const detailHeading = document.querySelector(".detail-view h1");
+const postureOutput = document.querySelector(".posture-output");
+
+list.addEventListener("click", (event) => {
+  detailHeading.textContent = event.target.textContent;
+});
+
+function reportPostureOutput() {
+  postureOutput.textContent = `Device posture: ${navigator.devicePosture.type}`;
```

Comment:
I can get `continuous` in Chrome `132.0.6834.160`, but I'm guessing there's no way to emulate folding is there? I can choose a foldable in dev tools, but I'm not sure if events are really passed through / created there

![Screenshot 2025-02-03 at 10 37 42](https://github.com/user-attachments/assets/42f17383-1040-4edf-94ca-bf6e9fb0edd8)

Reply:
> I got @darktears to test the demo on a physical device, and he said it was working correctly.

Righto! I don't have access to one, so I can't check, but I trust it works as expected